test-basic/query-builder.js

```diff
@@ -2001,7 +2001,50 @@ describe('query-builder', function() {
         {'false-query':null}
         );
   });
-
+  it('should create a before-query', function(){
+   var d = new String('foo');
+   assert.deepEqual(
+        q.before('foo'),
+        {'before-query':{timestamp:d}}, 'Error in Before-Query with String input'
+        );
+  });
+  it('should create a before-query', function(){
+   var d = new String(1562178870875);
```

Comment:
This test value is a string.  

A number would be either or (better) both of `var d = 1562178870875;` and `var d = new Number(1562178870875);`